config/initializers/acts_as_taggable_on.rb

```diff
@@ -1,2 +1,3 @@
 ActsAsTaggableOn.remove_unused_tags = true
 ActsAsTaggableOn.force_lowercase = true
+ActsAsTaggableOn::Tagging.belongs_to :taggable, polymorphic: true, touch: true
```

Comment:
By default I think the line is (where `taggable` is the group or resource record): 

```
ActsAsTaggableOn::Tagging.belongs_to :taggable, polymorphic: true
```

Here, I just override it to add the `touch: true` that will triggers the related record update.